doc/src/odin/dev_guide/regression_test.md

```diff
@@ -297,6 +297,10 @@ This regression test targets cases that require a lot of ram and time.
 
 The micro regression tests targets hards blocks and pieces that can be easily instantiated in architectures.
 
+### mixing_optimization
+
+The mixing optimization regression tests targets mixing implementations for operations implementable in hard blocks and their soft logic counterparts that can be can be easily instantiated in architectures.
+
 ### opertators
```

Comment:
can you fix this typo while you are changing this file?

if should say `operators`

Reply:
sure, thank you!